spynnaker/pyNN/models/pynn_population_common.py

```diff
@@ -182,6 +184,7 @@ def __init__(
             for variable, value in iteritems(initial_values):
                 self._initialize(variable, value)
 
+    # TODO DEAD?
```

Comment:
> `TODO DEAD?`

Reply:
fixed

Reply:
thanks, coz it made me chase up about the pop scope splitter object support